pkg/esoutput/mapping.json

```diff
@@ -19,10 +19,10 @@
             "enabled": true
         },
         "properties": {
-            "Time": {
```

Comment:
What's the reason for changing the mapping?

Reply:
by introducing elasticMetricEntry I renamed the fields to be distinguishable from any other entity. This was more a debugging or cosmetic issue while developing so I reverted it as it is technically unecesssary.